api/components.yaml

```diff
@@ -0,0 +1,81 @@
+swagger: '2.0'
+definitions:
+  Error:
```

Comment:
@gunzip Let me check https://github.com/zalando/problem. That could be the new standard ;)